packages/wonder-blocks-modal/src/themes/default.ts

```diff
@@ -1,36 +1,69 @@
-import * as tokens from "@khanacademy/wonder-blocks-tokens";
+import {
+    border,
+    semanticColor,
+    spacing,
+} from "@khanacademy/wonder-blocks-tokens";
 
 const theme = {
-    color: {
-        bg: {
-            inverse: tokens.color.darkBlue,
+    /**
+     * Shared tokens
+     */
+    root: {
+        // TODO(WB-1852): Remove light variant.
+        color: {
+            inverse: {
+                background: semanticColor.surface.inverse,
+                foreground: semanticColor.text.inverse,
+            },
         },
-        text: {
-            inverse: tokens.color.white,
-            secondary: tokens.color.offBlack64,
+        border: {
+            radius: border.radius.medium_4,
+            width: border.width.thin,
         },
-        shadow: {
-            default: tokens.color.offBlack16,
+    },
+    /**
+     * Building blocks
+     */
+    backdrop: {
+        color: {
+            background: semanticColor.surface.overlay,
+        },
+    },
+    dialog: {
+        spacing: {
+            padding: spacing.medium_16,
+        },
+    },
+    footer: {
+        color: {
+            border: semanticColor.border.primary,
         },
     },
-    border: {
-        radius: tokens.border.radius.medium_4,
-        width: tokens.border.width.thin,
-        color: tokens.color.blue,
+    header: {
+        color: {
+            border: semanticColor.border.primary,
+            secondary: semanticColor.text.secondary,
+        },
+        spacing: {
+            paddingBlockMd: spacing.large_24,
+            paddingInlineMd: spacing.xLarge_32,
+            paddingInlineSm: spacing.medium_16,
+            gap: spacing.xSmall_8,
+            titlePaddingRightMd: spacing.medium_16,
+            titlePaddingRightSm: spacing.xLarge_32,
+        },
+    },
+    panel: {
+        color: {
+            border: semanticColor.border.focus,
+        },
+        spacing: {
+            paddingBlockEnd: spacing.xLarge_32,
```

Comment:
For token names, what do you think about naming the tokens so that they aren't specific to how it is used in css? That way, if we were to change how we apply the spacing value in the component, we wouldn't have to rename the token!

Reply:
I think it makes sense in this case as it can be renamed to `gap` (not directly related to the css prop). In other cases, it might be trickier where tokens are directly associated to specific properties (e.g. the `header` padding).